.gitignore

```diff
@@ -3,3 +3,7 @@
 .packages
 .pub
 pubspec.lock
+
+# IntelliJ stuff
```

Comment:
Let's revert this file - we don't want editor specific stuff here. You might want to add these to a personal global gitignore:

https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer